nixos/modules/services/networking/nats.nix

```
@@ -22,8 +22,8 @@ let
           nativeBuildInputs = [ nats-server ];
         }
         ''
-          nats-server --config "${configFile}" -t
-          ln -s "${configFile}" "$out"
+          nats-server --config "${unvalidatedConfigFile}" -t
+          ln -s "${unvalidatedConfigFile}" "$out"
```

looking at the lambda itself, it seems to expect a file argument to be passed to it. so either we can use that or just remove it.
```suggestion
          nats-server --config "${file}" -t
          ln -s "${file}" "$out"
```